frontend/admin/src/js/components/user/UserTable.tsx

```diff
@@ -96,13 +83,43 @@ export const UserTable: React.FC = () => {
     col: string | undefined;
   }>();
 
+  // merge search bar input with fancy filter state
+  const filterCombiner = (
+    fancyFilterState: UserFilterInput | undefined,
+    searchBarTerm: string | undefined,
+    searchBarCol: string | undefined,
```

Comment:
"column" feels like a leaky abstraction from the table

```suggestion
    searchType: string | undefined,
```

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/commit/a2cb77df9d066a66a0701d8289082a4d0f68742f